src/MiNET/MiNET/Utils/IO/Compression.cs

```diff
@@ -32,26 +32,46 @@
 
 namespace MiNET.Utils.IO
 {
+	//TODO - rework on NONE and ZLib instances
```

Comment:
What is it you want to rework exactly?

Reply:
Split into two implementations: without compression and with ZLib compression.
Since now before sending McpeNetworkSettings we should not use compression. And it's not the same as ZLib "NoCompression"